benchexec/tools/svf_svc.py

```diff
@@ -40,23 +38,20 @@ def cmdline(self, executable, options, task, rlimits):
 
         if task.property_file:
             options += ["--prop", task.property_file]
+
+        if rlimits.cputime:
+            options += ["--time-limit", str(rlimits.cputime)]
+
         return [executable] + options + [task.single_input_file]
 
     def determine_result(self, run):
-        good_exit = False
-        verified = True
-
         for line in run.output:
             if line.startswith("Correct"):
-                good_exit |= True
-                verified &= True
+                return result.RESULT_TRUE_PROP
             elif line.startswith("Incorrect"):
-                good_exit |= True
-                verified &= False
-
-        if not good_exit:
-            return result.RESULT_UNKNOWN
-        elif verified:
-            return result.RESULT_TRUE_PROP
-        else:
-            return result.RESULT_FALSE_PROP
+                return result.RESULT_FALSE_PROP
+            elif line.startswith("Error"):
+                # The line should contain the error message with details.
+                return line
+
+        return result.RESULT_UNKNOWN
```

Comment:
Typically, if the tool outputs nothing or something completely unexpected, this is because of an error (for example if the tool crashes and dumps a stacktrace or segfaults or so). So I would recommend to return `RESULT_ERROR` here. If your tool gives up willingly because it does not know the answer, it would be nice for users as well if the output would indicate this, and this could be matched here for the `UNKNOWN` case.

Reply:
Done.